server/lib/nl/nl_variable.py

```diff
@@ -43,38 +57,82 @@ class VariableStore:
   entries: List[Entry]
 
 
-def expand_svg(svgs: List[str]):
-  """Expand svg"""
-  result = {}
-  svg2svs = dc.property_values(svgs, "memberOf", False)
-  # svgs_l2 is to search one more level
-  svgs_l2 = []
-  for svg, svs in svg2svs.items():
-    if not svs:
-      svgs_l2.append(svg)
-      result[svg] = []
+def parse_sv(sv_definition: str) -> SV:
+  res = SV()
+  parts = sv_definition.split(",")
+  for part in parts:
+    k, v = part.split("=")
+    if k == "pt":
+      res.pt = v
+    elif k == "mp":
+      res.mp = v
+    elif k == "st":
+      res.st = v
     else:
-      result[svg] = svs
-  if svgs_l2:
-    child_svgs = []
-    svg2childsvgs = dc.property_values(svgs_l2, "specializationOf", False)
-    all_child_svgs = []
-    for svg, child_svgs in svg2childsvgs.items():
-      all_child_svgs.extend(child_svgs)
-    childsvg2svs = dc.property_values(all_child_svgs, "memberOf", False)
-    for svg, child_svgs in svg2childsvgs.items():
-      for child_svg in child_svgs:
-        result[svg].extend(childsvg2svs[child_svg])
-  return result
-
-
-def extend_svs(svs: List[str]):
-  """Extend all svs with siblings svs under the same svg.
+      res.pvs[k] = v
+  return res
+
+
+def parse_svg(svg_dcid: str) -> SVG:
```

Comment:
please note that SVG DCID does not directly relate to the schema. We should probably set definitions for SVGs too in the cache?

Reply:
have definition for SVG is very helpful!